packages/infrastructure/authentik_atlas_mongodb_sso/README.md

```diff
@@ -0,0 +1,84 @@
+# Atlas MongoDB SSO with Authentik
+
+This module sets up the Authentik Application and Provider for Atlas MongoDB.
+
+## Sample Terragrunt Configuration
+
+```hcl
+include "panfactum" {
+  path   = find_in_parent_folders("panfactum.hcl")
+  expose = true
+}
+
+terraform {
+  source = include.panfactum.locals.pf_stack_source
+}
+
+dependency "authentik_core" {
+  config_path = "../authentik_core_resources"
+}
+
+dependency "kube_authentik" {
+  config_path = "../kube_authentik"
+}
+
+inputs = {
+  acs_url      = "https://auth.mongodb.com/sso/saml2/sample"
+  issuer       = "https://authentik.sample.com"
+  audience     = "https://www.okta.com/saml2/service-provider/sample"
+
+  organization_name   = dependency.authentik_core.outputs.organization_name
+  authentik_namespace = dependency.kube_authentik.outputs.namespace
+  media_configmap     = dependency.kube_authentik.outputs.media_configmap
+  authentik_domain    = dependency.kube_authentik.outputs.domain
+
+  allowed_groups = [
+    "superusers",
+    "privileged_engineers",
+    "engineers",
+    "restricted_engineers",
+    "billing_admins"
+  ]
+}
+```
+
+## Guide
+### Initialize the module
+1. Create a `authentik_atlas_mongodb_sso` module directory adjacent to the `authentik_core_resources` directory.
+2. Create a `terragrunt.hcl` file in the `authentik_atlas_mongodb_sso` directory and include the sample above.
+3. Replace the input `issuer` with your authentik domain url.
+4. run `pf-tf-init`
+5. run `terragrunt apply`
+
+### Download the Signing Certificate
+6. Login to your Authentik instance
+7. Go to the `Admin` section
+8. Click on `Applications` -> `Providers`
+9. Find `Related objects` and click on the `Download` button for the `Download signing certificate`
+10. Note the `SSO URL (POST)` string
+
+### Configure the Atlas MongoDB Identity Provider
```

Comment:
I am confused why we are listing manual steps here?

There is a terraform provider that can be used to automate this setup. See [docs](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_settings_identity_provider).

Ideally, I'd like to see these PRs have two modules: one for Authentik and one for the service provider.